MbDotNet/Models/Imposters/Imposter.cs

```diff
@@ -1,29 +1,35 @@
-﻿using MbDotNet.Enums;
+﻿using MbDotNet.Enums;
 using MbDotNet.Exceptions;
-using Newtonsoft.Json;
-
-namespace MbDotNet.Models.Imposters
-{
-    public abstract class Imposter
-    {
-        /// <summary>
-        /// The port the imposter is set up to accept requests on.
-        /// </summary>
-        [JsonProperty(PropertyName = "port", DefaultValueHandling = DefaultValueHandling.Ignore)]
-        public int Port { get; private set; }
-
-        /// <summary>
-        /// The protocol the imposter is set up to accept requests through.
-        /// </summary>
-        [JsonProperty("protocol")]
-        public string Protocol { get; private set; }
-
-        /// <summary>
-        /// Optional name for the imposter, used in the logs.
-        /// </summary>
-        [JsonProperty("name")]
-        public string Name { get; private set; }
-
+using Newtonsoft.Json;
+
+namespace MbDotNet.Models.Imposters
+{
+    public abstract class Imposter
+    {
+        /// <summary>
+        /// The port the imposter is set up to accept requests on.
+        /// </summary>
+        [JsonProperty(PropertyName = "port", DefaultValueHandling = DefaultValueHandling.Ignore)]
+        public int Port { get; private set; }
+
+        /// <summary>
+        /// The protocol the imposter is set up to accept requests through.
+        /// </summary>
+        [JsonProperty("protocol")]
+        public string Protocol { get; private set; }
+
+        /// <summary>
+        /// Optional name for the imposter, used in the logs.
+        /// </summary>
+        [JsonProperty("name")]
+        public string Name { get; private set; }
+
+        /// <summary>
+        /// Adds mock verification support by remembering the requests made to this imposter. Note that this represents a memory leak for any long running mb process, as requests are never forgotten.
+        /// </summary>
+        [JsonProperty("recordRequests")]
+        public bool RecordRequests { get; set; }

```

Comment:
Why was this added? I don't see this on the [imposters contract](http://www.mbtest.org/docs/api/contracts) and it doesn't appear to be used anywhere. Unless there is a different reason for it, please remove it.

Reply:
Was trying to make use of the new property - see http://www.mbtest.org/docs/protocols/http. Will double check

Reply:
Ah, I see. It must not have been added to that contracts page. I think I'd prefer if that was added in a separate pull request since it is not really related to the proxy changes.